fs/expose/dokan/__init__.py

```diff
@@ -7,20 +7,26 @@
 This module provides the necessary interfaces to mount an FS object into
 the local filesystem using Dokan on win32::
 
-    http://dokan-dev.net/en/
+    http://dokan-dev.github.io/
 
-For simple usage, the function 'mount' takes an FS object and a drive letter,
-and exposes the given FS as that drive::
+For simple usage, the function 'mount' takes an FS object
+and new device mount point or an existing empty folder
+and exposes the given FS as that path::
 
     >>> from fs.memoryfs import MemoryFS
     >>> from fs.expose import dokan
     >>> fs = MemoryFS()
-    >>> mp = dokan.mount(fs,"Q")
-    >>> mp.drive
-    'Q'
+    >>> # Mount device mount point
+    >>> mp = dokan.mount(fs,"Q:\\")
     >>> mp.path
     'Q:\\'
     >>> mp.unmount()
+    >>> fs = MemoryFS()
+    >>> # Mount in a exisiting empty folder
```

Comment:
Typo - should be "an existing" (extra `n`, one less `i`).
Did you want to add spaces after the commas in your mount examples?